scripts/set_compiler_env.sh

```
@@ -1,47 +1,72 @@
 
-dpkg --add-architecture $DEBIAN_ARCH
-apt-get update
-apt-get install -y git curl crossbuild-essential-${DEBIAN_ARCH} libc-dev:${DEBIAN_ARCH} autoconf automake libtool m4 pkg-config cmake dpkg-dev
+DEBIAN_ARCH="${TARGETARCH}"
+if [[ "${TARGETARCH}" != "amd64" && "${TARGETARCH}" != "arm64" ]]; then
+  echo "Warning: ${TARGETPLATFORM} is NOT supported. Just compile with the best efforts."
 
-dpkg-architecture -a $DEBIAN_ARCH >/env
-export $(cat /env)
+  # best efforts
+  if [[ "${TARGETARCH}" = "arm" ]]
```

Shellcheck (0.11.0)

[error] Did you forget the 'then' for this 'if'?

(SC1049)